mediacloud/mediawords/util/html.py

```diff
@@ -81,3 +88,107 @@ def __get_meta_refresh_url_from_tag(inner_tag: str, inner_base_url=None) -> Opti
             return url
 
     return None
+
+
+def _sententize_block_level_tags(s: str) -> str:
```

Comment:
"sententize"?  😄

Reply:
probably bad humor, but trying to get across the idea that I'm making the text more easily sentence parseable.  this used to be called something like 'add_new_lines_around_block_level_tags()', but that doesn't really express the purpose or even what it does.